Mathlib/Data/Finset/Lattice.lean

```diff
@@ -1090,9 +1087,9 @@ theorem _root_.map_finset_inf' [SemilatticeInf β] [InfHomClass F α β] (f : F)
 
 @[simp]
 theorem inf'_image [DecidableEq β] {s : Finset γ} {f : γ → β} (hs : (s.image f).Nonempty)
-    (g : β → α) (hs' : s.Nonempty := (Nonempty.image_iff _).1 hs) :
-    (s.image f).inf' hs g = s.inf' hs' (g ∘ f) :=
-  @sup'_image αᵒᵈ _ _ _ _ _ _ hs _ hs'
+    (g : β → α)  :
+    (s.image f).inf' hs g = s.inf' hs.of_image (g ∘ f) :=
+  @sup'_image αᵒᵈ _ _ _ _ _ _ hs _
```

Comment:
For rewriting, can we have the same lemma but turned around and with `(hs : s.Nonempty)`?

Reply:
Done. I hope, I didn't miss anything.